### PR TITLE
GML-1615 feedback collection and update endpoint

### PR DESCRIPTION
**Pull request by luzhoutg**

### CODE REVIEW

**Review by RobRossmiller-TG — CHANGES_REQUESTED** [review]
rm run script, please

File: chat-history/db/db.go [hunk 1]
```diff
@@ -98,11 +99,33 @@ func UpdateConversationById(message structs.Message) (*structs.Conversation, err
...
+			}
+		} else {
+			return nil, tx.Error
+		}
+	} else {
+		// Update only the feedback and comments fields if message exists
+		existingMessage.Feedback = message.Feedback
+		existingMessage.Comment = message.Comment
+
+		if result := db.Model(&existingMessage).Updates(map[string]any{
```
Comment (RobRossmiller-TG) [inline chat-history/db/db.go:118]: Don't use `map[string]any` if necessary.  We should be able to do something like [this](https://gorm.io/docs/update.html#Update-Selected-Fields:~:text=db.Model(%26user).Select(%22Name%22%2C%20%22Age%22).Updates(User%7BName%3A%20%22new_name%22%2C%20Age%3A%200%7D))

File: common/py_schemas/schemas.py [hunk 2]
```diff
@@ -122,3 +122,10 @@ class QueryDeleteRequest(BaseModel):
 class QueryUpsertRequest(BaseModel):
     id: Optional[str]
     query_info: Optional[GSQLQueryInfo]
+
+
+class Message(BaseModel):
+    conversation_id: str
+    message_id: str
+    feedback: Optional[str]
+    comment: Optional[str] = None
```
Comment (RobRossmiller-TG) [inline common/py_schemas/schemas.py:131]: Why does comment have a None default, but not feedback?

**Review by RobRossmiller-TG — CHANGES_REQUESTED** [review]
Couple comments/questions

File: common/py_schemas/schemas.py (hunk 2 above)
Comment (luzhoutg) [reply common/py_schemas/schemas.py:131]: sure. add default as None

> Why does comment have a None default, but not feedback?

File: chat-history/db/db.go (hunk 1 above)
Comment (luzhoutg) [reply chat-history/db/db.go]: if result := db.Model(&existingMessage).Select("Feedback", "Comment").Updates(
			structs.Message{
				Feedback: message.Feedback,
				Comment:  message.Comment,
			}); result.Error != nil {
			return nil, result.Error
		}
		
changed now

> Don't use `map[string]any` if necessary. We should be able to do something like [this](https://gorm.io/docs/update.html#Update-Selected-Fields:~:text=db.Model(%26user).Select(%22Name%22%2C%20%22Age%22).Updates(User%7BName%3A%20%22new_name%22%2C%20Age%3A%200%7D))

**Review by RobRossmiller-TG — APPROVED** [review]

**Pull request merged**